filebeat/module/haproxy/log/ingest/pipeline.json

```diff
@@ -21,6 +21,7 @@
         },
         {
             "date": {
+                "if": "ctx.event.timezone == null",
```

Comment:
hmm I don't think we need to check timezone here because we are not using it in this session.